install.sh

```diff
@@ -94,18 +74,26 @@ if [ $isNonInteractive = false ]; then
       nano .env
       ;;
   esac
-  # Select install type
-  log "Do you want to build Docker images from source or use pre-built images (recommended)? This saves significant build time but requires good download speeds for it to complete fast." $COLOR_RED  
-  select choice in "From source" "Use pre-built images"; do
-    case $choice in
-      "From source" )
-        INSTALL_TYPE="source"
-        break;;
-      "Use pre-built images" )
-        INSTALL_TYPE="prebuilt"
-        break;;
-    esac
-  done
+  # Select install type
+  log "Do you want to build Docker images from source or use pre-built images (recommended)? \nThis saves significant build time but requires good download speeds for it to complete fast." $COLOR_RED
+  log "1) From source" $COLOR_GREEN
+  log "2) Use pre-built images (default)" $COLOR_GREEN
+  read -p "Enter your choice (1 or 2, default is 2): " choice
+
+  case $choice in
+      1)
+          INSTALL_TYPE="source"
+          ;;
+      2|"")
+          INSTALL_TYPE="prebuilt"
```

Comment:
this also need the .env-dist and .env to be modified because it's a value part of the non interactive install
it's just a flag for the script, it does not have to be consistent
you can't just edit the var value.
I'm not agree modifying this